tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -118,25 +126,16 @@ def test_failed_sqs_invocation_call_throws_general_exception(mocker, event):
 
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_failed_alb_invocation_call_throws_general_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
+    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_retry_sqs')
     mocker.patch(f'{LAMBDA_MODULE}.process_body_from_alb_invocation', side_effect=Exception)
     response = vetext_incoming_forwarder_lambda_handler(event, None)
 
     assert response['statusCode'] == 500
     sqs_mock.assert_not_called()
 
-@pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
-def test_failed_alb_invocation_call_throws_key_error_exception(mocker, event):
-    sqs_mock = mocker.patch(f'{LAMBDA_MODULE}.push_to_sqs')
-    mocker.patch(f'{LAMBDA_MODULE}.process_body_from_alb_invocation', side_effect=KeyError)
-    response = vetext_incoming_forwarder_lambda_handler(event, None)
-
-    assert response['statusCode'] == 424
-    sqs_mock.assert_not_called()
-
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn), (sqsInvokedWithAddOn)])
 def test_failed_ssm_paramter_retrieval_returns_empty_string(mocker,  event):
```

Comment:
How is this function directly testing that a function returned the empty string?  Can you test by calling the function directly instead of the handler?

Reply:
poorly named test.  it's testing that if the ssm parameter returns empty string, then it dumps the message in the retry queue.